conversations.yaml

```diff
@@ -0,0 +1,536 @@
+openapi: 3.0.0
+info:
+  title: Conversations API
+  version: 1.0.0
+  description:
+    Medallia Conversations enables communications with customers and employees
+    through a conversational service such as SMS, Facebook Messenger, WhatsApp,
+    and more.  A channel adapter connects Medallia Conversations with the end
+    user device, handling the device-specific communications as needed.
+  termsOfService: https://medallia.com/about/legal/terms/api
+servers:
+  - url: 'https://{conversations-api-gateway-hostname}'
+    variables:
+      api-gateway-hostname:
+        default: conversations.sc4.medallia.com
+        description:
+          The Medallia Experience Cloud hostname used to access
+          Medallia Conversations' API gateway.
+  - url: 'https://{adapter-api-gateway-hostname}'
+    variables:
+      adapter-api-gateway-hostname:
+        default: adapter-gateway.example.com
+        description:
+          The endpoint used to make requests from Medallia Conversations
+          to the channel adapter.
+paths:
+  /:
+    servers:
+      - url: 'https://{adapter-api-gateway-hostname}'
+        variables:
+          adapter-api-gateway-hostname:
+            default: adapter-gateway.example.com
+            description:
+              The endpoint used to make requests from Medallia
+              Conversations to the channel adapter.
+    post:
+      tags:
+        - Channel Adapter Services
+      description:
+        The API endpoint implemented by the Channel Adapter to accept
+        messages from Medallia Conversations.
+      parameters:
+        - in: query
+          name: access_token
+          schema:
+            type: string
+          required: true
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/OutboundMessageRequest'
+      responses:
+        200:
+          description: A successful response includes the customer and message IDs
+          content:
+              application/json:
+                schema:
+                  $ref: '#/components/schemas/OutboundMessageResponse'
```

Comment:
Can the channel adapter respond with any other response code?  What if the `access_token` is incorrect/expired, for example?